src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/probationoffendersearchapi/IDs.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.model.probationoffendersearchapi
+
+data class IDs(
+	val crn: String,
+	val pncNumber: String? = null,
+	val croNumber: String? = null,
+	val niNumber: String? = null,
+	val nomsNumber: String? = null,
+	val immigrationNumber: String? = null,
+	val mostRecentPrisonerNumber: String? = null,
+	val previousCrn: String? = null,
```

Comment:
did you intend to use this indentation spacing?

Reply:
ah thanks, fixed now. I really need a session on how to auto-format locally next week 🙃